src/main/java/com/squareup/kotlinpoet/ParameterizedTypeName.kt

```diff
@@ -20,10 +20,7 @@ package com.squareup.kotlinpoet
 import java.lang.reflect.Modifier
 import java.lang.reflect.ParameterizedType
 import java.lang.reflect.Type
-import kotlin.reflect.KClass
-import kotlin.reflect.KType
-import kotlin.reflect.KTypeParameter
-import kotlin.reflect.KTypeProjection
+import kotlin.reflect.*
```

Comment:
Please revert. We do not allow star imports.

Reply:
Fixed